### ISSUE TITLE
#-432443104------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#493]
<strong>Source:</strong>`<a id="edit-button-31b05d32634046dca7dd571b032a08f4" href="/layers/31b05d32634046dca7dd571b032a08f4/edit" aria-describedby="calcite-tooltip-071fec10-c1ae-8d4f-c8b8-ab36597b6cd6">`
<strong>Element Location: </strong>#edit-button-31b05d32634046dca7dd571b032a08f4
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@229514ff
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
